cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -118,6 +118,15 @@ typedef enum {
 } uart_txpad_t;
 
 /**
+ * @brief   Available SERCOM UART flag selections
+ */
+typedef enum {
+    UART_NONE            = (0x0),         /**< No flags set */
+    UART_RUN_STANDBY     = (0x1 << 0),    /**< run SERCOM in standby mode */
```

Comment:
either use shift ~~of~~ or hex, but IMO no mix, i.e., use `0x0`, `0x1`, and `0x2` **or** `0`, `1 << 1`, and `1 << 2`. Further , it might be nice to prefix this enum vals with `UART_FLAG_*` instead of `UART_` only - just to be safe and avoid possible name clashes.

Reply:
done